tests/integration-tests/tests/update/test_update.py

```diff
@@ -490,6 +495,40 @@ def _check_role_attached_policy(region, cluster, policy_arn):
     assert policy_arn in policies
 
 
+def get_ebs_volume_ids(cluster, region):
+    # get the list of configured ebs volume ids
+    # example output: ['vol-000', 'vol-001', 'vol-002']
+    ebs_stack = utils.get_substacks(cluster.cfn_name, region=region, sub_stack_name="EBSCfnStack")[0]
+    return utils.retrieve_cfn_outputs(ebs_stack, region).get("Volumeids").split(",")
+
+
+def describe_volume(volume_id, region):
```

Comment:
In general, I would avoid using function names that are similar to another function or something else you might be calling.

`describe_volume` looks a little too similar to `boto3.client("ec2", region_name=region).describe_volumes`, and we might get confused when debugging or looking through the code.

How about something like `get_ebs_volume_info` instead?

Reply:
Good idea! Done